bot/scheduler.go

```diff
@@ -0,0 +1,45 @@
+package bot
+
+import (
+	"strconv"
+	"time"
+
+	"github.com/bobheadxi/calories/server"
+)
+
+var zoneDir string
+
+// Scheduler :
+type Scheduler struct {
+	server *server.Server
+}
+
+//NewScheduler : constructor
+func NewScheduler(sv *server.Server) *Scheduler {
+	sch := Scheduler{
+		server: sv,
+	}
+	//s := Scheduler{}
+	go sch.Start()
+	return &sch
+}
+
+// Start : start and check every hour to send scheduled summaries to users
+func (sch *Scheduler) Start() {
+
+	t := time.NewTicker(time.Hour * 1)
+	for {
+		sch.SendSummary()
+		<-t.C
+	}
+}
+
+// SendSummary : appropriately send all users the scheduled summaries depending on their timezone
+func (sch *Scheduler) SendSummary() {
```

Comment:
Again, this does not need to be part of a separate class, and it does not need to be public either